### PR TITLE
Run imports in parallel

### LINKED ISSUE #329: Run imports in parallel
[Today we're running all imports sequentially](https://github.com/gitpod-io/observability/blob/024eb1a6e5cd84afcadf88ec27ebc564e66492e2/installer/cmd/render.go#L163-L171), we could use goroutines to speed up the process

### PR DESCRIPTION
**Pull request by ArthurSens**
Fixes #329

### CODE REVIEW

File: installer/cmd/render.go [hunk 1]
```diff
@@ -296,3 +285,62 @@ func init() {
 		os.Exit(1)
 	}
 }
+
+// runImports will import all manifests declared in the import interface using parallelism
+func runImports(kImports []importer.KustomizeImporter, yImports []importer.YAMLImporter) (imports []string, err error) {
+	errc := make(chan error)
+	defer close(errc)
+	var errWg sync.WaitGroup
```
Comment (liam-j-bennett) [inline installer/cmd/render.go]: The x sync library has better groups for this: https://pkg.go.dev/golang.org/x/sync/errgroup

We should be able to make these loops with this `errgroup` package instead, waiting on the whole group. Hopefully, this should make the logic a bit simpler.

Comment (ArthurSens) [reply installer/cmd/render.go]: Interesting, I wasn't aware of this package! Let me try :)

Comment (ArthurSens) [reply installer/cmd/render.go]: Indeed code looks a lot cleaner!

**Review by liam-j-bennett — APPROVED** [review]
🚀

**Pull request merged**